README.md

```diff
@@ -14,23 +14,25 @@ Our crates:
 
 | crate | description |
 |-|-|
-| `blockchain` | chain structures and syncing functionality |
-| `crypto` | verification and signature definitions |
-| `encoding` | used for encoding and decoding |
+| `blockchain` | Chain structures and syncing functionality |
+| `crypto` | Verification and signature definitions |
+| `encoding` | Forest encoding and decoding |
 | `ipld` | IPLD data model for content-addressable data |
```

Comment:
Should we also reference AMT/HAMT structure here? Just a thought.

Reply:
I don't think we need to get into specifics of each structure